src/main/java/frc/robot/constants/RobotComponents.java

```diff
@@ -109,7 +109,7 @@
                 inverted(true).
                 withOpenLoopRampRate(0.5).
                 withClosedLoopRampRate(0.5).
-                withPID(new PIDFCoefs(0, 0, 0, 0, 0, 0, 0));
+                withPID(RobotConstants.LOCAL_CONSTANTS.localIntakeOpenerConstants.pidfCoefs);
```

Comment:
shorter import

Reply:
Lavi here: The only way is a static import to LOCAL_CONSTANT, which is both ugly, and not that helpful. It only gets rid of "RobotConstants.".